delta-api/src/main/java/io/cdap/delta/api/DeltaTargetContext.java

```diff
@@ -27,15 +27,15 @@
    * Record that a DML operation was processed. Metrics on the number of operations processed are stored in memory
    * and actually written out when {@link #commitOffset(Offset, long)} is called.
    *
-   * @param op type of DML operation
+   * @param op represents DML operation
```

Comment:
nit: no need to add "represents" , implicitly the comments here is to explain what does the param represent. Also you commented in a different way for DDL Operation param below (which I prefer that way), let's keep them consistent :)

Reply:
fixed.